pydantic/utils.py

```diff
@@ -134,7 +135,7 @@ def import_string(dotted_path: str) -> Any:
         raise ImportError(f'Module "{module_path}" does not define a "{class_name}" attribute') from e
 
 
-def truncate(v: str, *, max_len: int = 80) -> str:
+def truncate(v: Union[str], *, max_len: int = 80) -> str:
```

Comment:
Same as above, needed to use Union[str] here to prevent Cython from causing additional errors